### PR TITLE
Make error box clickable

### PR DESCRIPTION
**Pull request by Mnosek**
cc @tomalec @warpech

### CODE REVIEW

File: puppet-client.html [hunk 1]
```diff
@@ -68,10 +68,14 @@
...
             }
+            .reload {
+              display: block;
+              cursor: pointer;
+            }
         </style>
         <div class="box-container" hidden$="{{!showReconnectingIn}}"><div class="box notice"><strong>Not connected.</strong> Connecting in {{reconnectionSeconds}}s... <a href="" on-tap="reconnectNow">Retry now</a></div></div>
         <div class="box-container" hidden$="{{!showReconnectingNow}}"><div class="box notice"><strong>Connecting now...</strong></div></div>
-        <div class="box-container" hidden$="{{!showConnectionError}}"><div class="box error"><strong>Connection error.</strong> See console for details. <a href="" on-tap="reload">Reload page</a> </div></div>
+        <div class="box-container" hidden$="{{!showConnectionError}}"><a class="box error reload" on-tap="reload"><strong>Connection error.</strong> See console for details. <u>Click here to reload</u></a></div>
```
Comment (warpech) [inline puppet-client.html]: Please don't use the `u` element. It is not intended for underlined text, but something much weirder: http://html5doctor.com/u-element/

Use `span` instead.

File: test/d-b-n_smoke/reconnection.html [hunk 2]
```diff
@@ -63,8 +66,15 @@
...
+            });
+
+            it('and log error reason to the console', function() {
+                var closeEvent = {reason: 'thereason', code: 1};
                 puppet.network._ws.close(closeEvent);
-                expect(container.querySelector("div.box.error").innerText).to.contain(closeEvent.reason);
+                expect(consoleErrorSpy.getCall(0).args[0]).to.contain(closeEvent.reason);
             });
 
             describe('when connection is broken without a reason should display a notice box', function() {
```
Comment (warpech) [inline test/d-b-n_smoke/reconnection.html]: I was not able to run `wct` locally (seems to freeze for me), but running `http://localhost:8000/components/puppet-polymer-client/test/` in web browsers (Chrome, Firefox, Edge) shows 100% passing tests 👍

**Comment by warpech** [comment]
Ping. @Mnosek can you fix this?

**Comment by Mnosek** [comment]
Fixed!

**Comment by tomalec** [comment]
Could you fix the tests as well https://travis-ci.org/
```
timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
```
on all browsers seems as something fixable.
You can disable Edge until SauceLabs will update it.

**Pull request merged**